recipes/sqcircuit/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "SQcircuit" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 9627fb5bdce35018559beb5153ba6c17d6e4529c6a4152b1c07665e1e072d21f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

This looks like it's pure Python and could be made noarch:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```